pages/linux/update-alternatives.md

```diff
@@ -0,0 +1,19 @@
+# update-alternatives
+
+> A convenient tool for maintaining symbolic links that determining default commands.
```

Comment:
Umm the grammar got messed up a bit. I think @sbrl was trying to say "that determines". I actually prefer "to determine".

What do you say @sbrl ?

Reply:
Hrm, it has a bit! Yeah, it's a rather awkward situation here actually. I'd be happy with either "that determine" or "to determine".